src/main/java/bio/terra/service/snapshotbuilder/utils/AggregateSynapseQueryResultsUtils.java

```diff
@@ -19,4 +19,15 @@ public static SnapshotBuilderConcept aggregateConceptResult(ResultSet rs) {
           "Error processing result set into SnapshotBuilderConcept model", e);
     }
   }
+
+  public static int rollupCountsMapper(ResultSet rs) {
+    try {
+      // Azure ResultSet is 1 indexed
+      // https://docs.oracle.com/javase/7/docs/api/java/sql/ResultSet.html
+      return rs.getInt(1);
```

Comment:
If you want to avoid indexing by one you could also lookup the column using its alias.
```suggestion
      return rs.getInt("count");
```
Although if you do that you might want to use a named constant